Documentation/compatibility/winforms-accessibility-changes-471.md

```diff
@@ -46,26 +46,41 @@ For an overview of UI automation, see the [UI Automation Overview](https://docs.
 
 __Added support for UI Automation patterns and properties__
 
```

Comment:
It occurs to me that it might be good to add the following at the end of line 29:
These accessibility changes are enabled by default on Windows Forms applications that target the .NET Framework 4.7.1 or later.